client/src/Components/SharedComponents/StarRating.jsx

```diff
@@ -45,7 +45,7 @@ const StarRating = (props) => {
       // Multiply each data[rating] by 1 turns it into a number (it is originally type string)
       setAverageRating(rating / (data["1"] * 1 + data["2"] * 1 + data["3"] * 1 + data["4"] * 1 + data["5"] * 1));
     } else if(typeof data ==='number') {
-      setAverageRating(data);
+      setAverageRating(data)
```

Comment:
May I ask: why did you remove the semicolon here? I do not mean to be nit-picky or anything-- it shouldn't change anything and the app should still render and function fine. I am just wondering if there was a reason behind this.

Reply:
@Panlord I believe I deleted by accident but I plan on going back with a linter to cover these. Thanks for the heads up though!